projects/ngrx.io/content/guide/entity/install.md

```diff
@@ -0,0 +1,18 @@
+# Installation
```

Comment:
We also have a ng-add command for entity.

Reply:
Ok! I tried running but I get an error that the command doesn’t exist. Is it in a newer version than current?